### DISCUSSION TITLE
Will tevm ever support Arbitrum?

### DISCUSSION THREAD

**Discussion by 0xTARC** [evmts/tevm-monorepo#1395] [General]
Blocker for usage personally. Any ideas or plans for this on the horizon?

It seems unlikely to happen unless ethereumjs implements Arbitrum but a boy can dream.

**Comment by 0xTARC** [comment]
nvm am noob

**Comment by roninjin10** [comment]
Yea you can use arbitrum.

**Comment by roninjin10** [comment]
Correction: You can't use arbitrum because arbitrum deposit tx are not supported but that is being fixed in todays release
